ddtrace/contrib/unittest/patch.py

```diff
@@ -516,6 +683,20 @@ def _start_test_module_span(instance) -> ddtrace.Span:
     test_module_span.set_tag_str(test.TEST_TYPE, SpanTypes.TEST)
     test_module_span.set_tag_str(test.MODULE, test_module_name)
     test_module_span.set_tag_str(test.MODULE_PATH, _extract_module_file_path(instance))
+    test_module_span.set_tag_str(
+        test.ITR_TEST_CODE_COVERAGE_ENABLED,
+        "true" if _CIVisibility._instance._collect_coverage_enabled else "false",
+    )
+    if _CIVisibility.test_skipping_enabled():
+        test_module_span.set_tag_str(test.ITR_TEST_SKIPPING_ENABLED, "true")
+        test_module_span.set_tag(test.ITR_TEST_SKIPPING_TYPE, TEST)
+        test_module_span.set_tag_str(test.ITR_TEST_SKIPPING_TESTS_SKIPPED, "false")
+        test_module_span.set_tag_str(test.ITR_DD_CI_ITR_TESTS_SKIPPED, "false")
+        test_module_span.set_tag_str(test.ITR_FORCED_RUN, "false")
+        test_module_span.set_tag_str(test.ITR_UNSKIPPABLE, "false")
+        test_module_span.set_metric(test.ITR_TEST_SKIPPING_COUNT, 0)
```

Comment:
Feels like these `set_tag`/`set_tag_str` should match the ones above (eg: https://github.com/DataDog/dd-trace-py/pull/7125/files#diff-34d77150382cf4bb09fce56b5ae8364e0f3293de8791431903a983dc80f5e63fR646 uses `set_tag()` for `test.ITR_TEST_SKIPPING_TESTS_SKIPPED` while https://github.com/DataDog/dd-trace-py/pull/7125/files#diff-34d77150382cf4bb09fce56b5ae8364e0f3293de8791431903a983dc80f5e63fR693 uses `sset_tag_str()`).